package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "node_ecourse",
+  "version": "0.0.1",
+  "description": "Node.js course tasks",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel ./app/server/ --out-dir dist",
+    "start": "babel ./app/server/ --watch --out-dir dist --source-maps inline",
```

Comment:
If you want  to have a script for development only, you could try [`babel-watch`](https://www.npmjs.com/package/babel-watch), which runs on the fly, without compile the code to a directory.

Reply:
Done